Initial commit

beechtom opened the pull request:
Part of puppetlabs/bolt#1707

beechtom committed:
Initial commit

beechtom commented:
Something got messed up with the git history and the previous PR for this was auto-closed: https://github.com/puppetlabs/puppetlabs-gcloud_inventory/pull/3

nicklewis commented on README.md:
```diff
@@ -0,0 +1,71 @@
+
+| Option | Type | Description |
+| ------ | ---- | ----------- |
+| `credentials` | `String` | An absolute path to the service account credentials file. _Optional._ |
```
Should this be absolute or should it be relative to the Boltdir (which would still allow absolute)?

beechtom replied:
Do we have access to the Boltdir path from a task?

nicklewis replied:
We do from plugins. It's passed as the `_boltdir` param.

https://github.com/puppetlabs/puppetlabs-terraform/blob/master/tasks/resolve_reference.rb#L48

beechtom replied:
Ah, nice. I think it makes sense to have it be relative in that case.

beechtom committed:
Expand credentials file path relative to Boltdir

This updates the `credentials` option so it expands relative to the Bolt
project directory.

nicklewis commented:
I don't seem to have write access to this repo to merge this PR...

beechtom commented:
@nicklewis Added @puppetlabs/bolt as admin.
